Core/IFC/DatabaseIFC.cs

```diff
@@ -287,12 +288,48 @@ public bool WriteFile(string filename)
 #endif
 				sw = new StreamWriter(FileName);
 			bool result = new SerializationIfcSTEP(this).WriteSTEP(sw, FileName);
+			sw.Close();
 #if (!NOIFCZIP)
 			if (zip)
 				za.Dispose();
 #endif
 			return result;
 		}
+		public bool WriteStream(Stream stream, string filename)
```

Comment:
Thanks for the pull request.  Do you think this method should take an FormatIfcSerialization enumeration rather than a filename?  If it's a stream, there might be no filename as I understand it.

Reply:
My thougt was mainly to assemble the STEP file header, but I didn't saw there is a `Filename` property from `DatabaseSTEP` which could be used.
When I change to `FormatIfcSerialization` how should I handle the .ifczip?